lib/i18n_sonder/upload_source_strings.rb

```diff
@@ -0,0 +1,23 @@
+module I18nSonder
+  class UploadSourceStrings
+    include UploadHelpers
+
+    def perform(model, options = {})
+      # TOOD: Don't need to check if shoud_upload_for_translation?
+      # since most of the time can assume this is triggered when a unit is first added to
+      # return unless should_upload_for_translation?(model, locale, value, attribute)
```

Comment:
This may be correct from the perspective of unit/listing, but since this is a generic library to be used in any case for dynamic content I think we should be defensive and still apply the `should_upload_for_translation?` check. It's one extra call and logic check, but I don't think that's too much of a burden. What do you think?

Reply:
Ah forgot its such a generic library and not tailored to flatbook, being defensive makes sense and I don't think its much of a burden either